src/gc/gcsvr.cpp

```diff
@@ -4,6 +4,7 @@
 
 
 
+#define OS_PAGE_SIZE 0x1000
 #include "common.h"
```

Comment:
This creates two definitions for OS_PAGE_SIZE in the same project.  Needs to be actual current OS_PAGE_SIZE.